pkg/asset/ignition/bootstrap/content/report.go

```diff
@@ -0,0 +1,56 @@
+package content
+
+const (
+	// ReportSystemdContents is a service that reports the bootstrap progress
+	// via a Kubernetes Event.
+	ReportSystemdContents = `
+[Unit]
+Description=Report the completion of the cluster bootstrap process
+# Workaround for https://github.com/systemd/systemd/issues/1312
+Wants=bootkube.service tectonic.service
+After=bootkube.service tectonic.service
```

Comment:
Because of https://github.com/systemd/systemd/issues/1312, it is possible for `tectonic.service` and `bootkube.service` to run simultaneously (e.g. if `bootkube.service` fails and then retries). I'm not sure if `tectonic.service` can finish before `bootkube.service`, but this was easy and robust against future changes.

I like the idea of being able to reuse that "report-progress" script in the services. I can make that change.

Reply:
I made the event name and message configurable, but we still need the breadcrumbs to sync on the state of both `bootkube.service` and `tectonic.service`. Otherwise, the installer would have to look for both events when determining whether or not to destroy the bootstrap node. I'm not sure the two events on their own are useful though, so I left that reporting out of this PR.